woocommerce/woocommerce-gutenberg-products-block
# Include issue submission link in the footer of all documention (.md files)

This PR provides 2 improvements to our docs.

1. Added links to all single routes on the main readme.md file for the Store API. Fixes #4473
2. Added a link to submit feedback/issues with docs in the footer of all .md files under ./docs ./package/checkout and src/StoreApi.
    - This includes a link with a prefilled issue title referencing the doc
    - The footer is generated automatically by a script; `npm run build:docs` will add or update the text across all markdown files.
    - I've ran this to generate the footers across all current docs.

Fixes #4473
Fixes #4452

### Testing

How to test the changes in this Pull Request:

1. For any doc under docs/, remove the footer text which contains the hiring message and issue submission link
2. Run `npm run build:docs`
3. Confirm the text is added back
4. Confirm the link for the submit issue contains a title referencing the current markdown file
